scripts/ldr

```diff
@@ -0,0 +1,177 @@
+    "start")
+      roachprod sql $A:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+      roachprod sql $B:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+      roachprod sql $A:1 -- -e "USE ycsb; SELECT crdb_internal.start_logical_replication_job($(roachprod pgurl --insecure $B:1), ARRAY['usertable']);"
+      roachprod sql $B:1 -- -e "USE ycsb; SELECT crdb_internal.start_logical_replication_job($(roachprod pgurl --insecure $A:1), ARRAY['usertable']);"
+      ;;
+    "pause")
+      roachprod sql $A:1 -- -e "PAUSE JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running');"
+      roachprod sql $B:1 -- -e "PAUSE JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running');"
+      ;;
+    "resume")
+      roachprod sql $A:1 -- -e "RESUME JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'paused');"
+      roachprod sql $B:1 -- -e "RESUME JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'paused');"
+      ;;
+    "cancel")
+      roachprod sql $A:1 -- -e "CANCEL JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running' OR status = 'paused');"
+      roachprod sql $B:1 -- -e "CANCEL JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running' OR status = 'paused');"
+      ;;
+    "stats")
+      roachprod sql $A -- -e "select * from crdb_internal.cluster_replication_node_streams;"  --format=table
+      roachprod sql $B -- -e "select * from crdb_internal.cluster_replication_node_streams;"  --format=table
+      ;;
+    *)
+      echo "unknown command '$1'; useage: $0 {start|pause|resume|cancel|stats}"
+      exit 1
+      ;;
+    esac
+    ;;
+
+  "workload")
+    shift
+    case "${1:-}" in
+      "init")
+        roachprod run $A:1 "./workload init ycsb --drop --families=false --splits 1000 --seed 1665361151361545632 $(roachprod pgurl --insecure $A)"
+        roachprod run $B:1 "./workload init ycsb --drop --families=false --splits 1000 --seed 665361151356328 $(roachprod pgurl --insecure $B)"
+        roachprod sql $A:1 -- -e "ALTER TABLE ycsb.public.usertable ADD COLUMN crdb_internal_origin_timestamp DECIMAL NOT VISIBLE DEFAULT NULL ON UPDATE NULL"
+        roachprod sql $B:1 -- -e "ALTER TABLE ycsb.public.usertable ADD COLUMN crdb_internal_origin_timestamp DECIMAL NOT VISIBLE DEFAULT NULL ON UPDATE NULL"
+        ;;
+      "run")
+        OUTPUT_FILE_A="a-ycsb-$(date '+%Y-%m-%d-%H:%M:%S').log"
+        OUTPUT_FILE_B="b-ycsb-$(date '+%Y-%m-%d-%H:%M:%S').log"
+        shift
+        roachprod run $A:1 "nohup ./workload run ycsb --tolerate-errors --workload='F' --families=false --duration=24h --concurrency=100 --seed 1665361151361545632 --ramp=5s $@ $(roachprod pgurl $A) > $OUTPUT_FILE_A 2> $OUTPUT_FILE_A &" &
```

Comment:
Forgot to hit send on this comment before finishing my initial review.  A few points here:

* IMO, the canonical workload should be 100% inserts, since that'll give us the most grief.  To achieve that, you'll want to go with these flags: 

`--workload='custom' --insert-freq=1.0 --insert-start=1000000`

and

`--workload='custom' --insert-freq=1.0 --insert-start=4611686018427387904`

* I also added this comment locally, which it might be nice to add in here:

```
    # Split the 2^63 key space in half and start at 4611686018427387904. This
    # leaves us significant space in both halfs if we want to start a third or
    # fourth cluster for n-way replication
```

* For this workload, you'll probably want to drop concurrency to something like 16, which on my cluster chews up 75%+ of the CPU.